completions/cvs

```diff
@@ -20,24 +20,25 @@ _comp_cmd_cvs__modules()
     fi
 }
 
-# TODO:API: rename per conventions, rework to use vars rather than outputting
-_cvs_commands()
+_comp_cmd_cvs__compgen_commands()
 {
-    cvs --help-commands 2>&1 | awk '/^(     *|\t)/ { print $1 }'
+    _comp_compgen_split -- "$(
+        "$1" --help-commands 2>&1 | awk '/^(     *|\t)/ { print $1 }'
+    )"
 }
 
-_comp_cmd_cvs__command_options()
+_comp_cmd_cvs__compgen_command_options()
 {
     _comp_compgen_help -- --help "$2"
 }
 
-_comp_cmd_cvs__kflags()
+_comp_cmd_cvs__compgen_kflags()
 {
     _comp_compgen -- -W 'kv kvl k o b v'
 }
 
 # @since 2.12
-_comp_xfunc_cvs_roots()
+_comp_xfunc_cvs_compgen_roots()
 {
     local -a cvsroots=()
```

Comment:
```suggestion
    local -a _cvsroots=()
```

The later reference of the array `cvsroots` should also be renamed.

When a function is updated to be a generator, the local variable should be renamed to avoid conflicts with the variable name possibly specified by `-v var`. The variable starting with `_` is reserved for internal use (i.e., _comp_compgen does not accept a variable name `_*` with the option `-v`).

Reply:
Move to #1028 (ff928a4e).